hs628_tools/Visualization.py

```diff
@@ -0,0 +1,294 @@
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+import pandas as pd
```

Comment:
Temporarily using pandas to simpliy reading in data for testing could be ok, but as discussed in the written assignment and in class, the final module was not supposed to rely on pandas.